Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [workspace]
 resolver = "2"
 members = ["sdk", "examples/*"]
+# We exclude the storage example from the workspace so that the dioxus features for other renderers are not enabled
```

Comment:
I am confused by this, why would enabling X feature in a specific example end up enabling it for the rest of crates in the workspace? Are you 100% sure?

Reply:
Or is it because you are using dioxus from git?

Reply:
It has been a while since I changed the workspace... I thought there was something weird going on with feature unification, but it seems like the storage example works in the workspace with the stable version of dioxus now, so I reverted these changes